plugins/elanfp/fu-elanfp-device.c

```diff
@@ -28,6 +28,21 @@
 #define REQTYPE_GET_VERSION 0xC1
 #define REQTYPE_COMMAND	    0x41
 
+#define TAG_SEND_COMMAND	0xFA
+
+/**
+ * FU_ELAN_FP_DEVICE_FLAG_USB_CONTROL_TRANSFER:
+ *
+ * Use usb control transfer.
+ */
+#define FU_ELAN_FP_DEVICE_FLAG_USB_CONTROL_TRANSFER (1 << 0)
+/**
+ * FU_ELAN_FP_DEVICE_FLAG_USB_BULK_TRANSFER:
+ *
+ * Use usb bulk transfer.
+ */
+#define FU_ELAN_FP_DEVICE_FLAG_USB_BULK_TRANSFER (1 << 1)
+
```

Comment:
why both?

Reply:
Old protocol is using a control transfer to send the command, the new protocol is using bulk transfer. Did you mean we only just to set a flag to new device with new protocol?

Reply:
> Did you mean we only just to set a flag to new device with new protocol

Reply:
Fixed in 7ee92a89640014327469a87cc9be9a48998b484c.